src/sidebar/components/test/annotation-test.js

```diff
@@ -834,14 +834,35 @@ describe('annotation', function() {
       });
 
       it('returns the value of the `flag_action` feature flag', function () {
-        var controller = createDirective().controller;
+        var ann = fixtures.defaultAnnotation();
+        var controller = createDirective(ann).controller;
+
+        ann.user = 'acct:notCurrentUser@localhost';
```

Comment:
The annotation should be fully initialised before being passed to `createDirective` (ie. move the `ann.user` line before `createDirective`). This happens to work because the component doesn't copy the annotation internally, but that isn't guaranteed.